drafts/liberty-dev-mode-vscode.adoc

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Open Liberty dev mode with VS Code"
+categories: blog
+author_picture: https://avatars2.githubusercontent.com/u/29640130
+author_github: https://github.com/ericglau
+seo-title: Open Liberty dev mode with VS Code - OpenLiberty.io
+seo-description: Start Open Liberty in development mode with a VS Code extension. Make changes to your application while the server is up, run tests and view results, and even debug the application without leaving the editor.
+blog_description: "Start Open Liberty in development mode with a VS Code extension. Make changes to your application while the server is up, run tests and view results, and even debug the application without leaving the editor."
+---
+= Open Liberty dev mode with VS Code
+Eric Lau <https://github.com/ericglau>
+
+The https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Liberty Dev VS Code extension] lets you start Open Liberty in development mode, make changes to your application while the server is up, run tests and view results, and even debug the application without leaving the editor.
+
+We've shown how link:/blog/2019/10/22/liberty-dev-mode.html[Open Liberty dev mode can be run from a command line] while allowing you to edit your code with any text editor or IDE.  In this blog post, you'll see how the optional extension for VS Code makes it even easier to interact with dev mode for a more seamless experience.
+
+For a demo of the extension, check out the following video:
+
+++++
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6wJRxUGdNRM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+++++
+
+== Installing the extension
+
+Install the https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Liberty Dev extension from the Visual Studio Marketplace].
+
+== Setting up the project
+
+For Maven projects, dev mode requires the Liberty Maven Plugin.  Specify the Liberty Maven Plugin with version `3.1` or above in your project's `pom.xml` file.
+[source,xml]
+----
+    <plugin>
+        <groupId>io.openliberty.tools</groupId>
+        <artifactId>liberty-maven-plugin</artifactId>
+        <version>3.1</version>
+    </plugin>
+----
+
+== What you can do with dev mode
```

Comment:
May be move this section to the bottom?

IOW:
- We introduce the extension
- How you can setup the extension and the project
- How can you interact with it from the menu
- What you can do with dev mode in VS Code with the extension?

Reply:
Moved the section as suggested.